doc/changelogs/CHANGELOG_V10.md

```diff
@@ -33,15 +33,18 @@
 
 * Assert
   * Calling `assert.fail()` with more than one argument is deprecated. [[`70dcacd710`](https://github.com/nodejs/node/commit/70dcacd710)]
-  * Calling `assert.ok()` with no arguments will now throw. [[`3cd7977a42`](https://github.com/nodejs/node/commit/3cd7977a42)]
```

Comment:
Should this deletion be explained?

Reply:
> Should this deletion be explained?

@vsemozhetbyt It's incorrect. `assert.ok()` always threw if there was no argument (probalby because `undefined` is false-y.)  What's changed is that there is a better message in that situation. While a nice change, I suspect it does not warrant calling out in the Notable Changes section.